testGameSearch/ContentView.swift

```diff
@@ -23,9 +23,10 @@ struct ContentView: View {
             self.$gameVM.gameName) {
                 self.gameVM.searchGames()
             }
-            
-            Text(self.gameVM.gameName)
-            
+            if self.gameVM.game != nil {
+                Text(self.gameVM.game!.name)
+                Text(self.gameVM.game!.summary ?? "")
+            }
```

Comment:
Just added some UI so you can see if you got a result back. force unwrap is needed here until swiftui gets if let support inside viewbuilder closures